docs/security/additional-security-features/introduction-to-additional-security-features.md

```diff
@@ -1,29 +1,23 @@
 
-Cloud Infrastructure Security is a cloud-native, enterprise-grade solution composed of security apps, customizable dashboards, and tools to analyze your security data. It leverages Sumo Logic's core functionality, including data collection, ingestion, and storage, to produce findings that help protect your attack surfaces from threats.
+[Cloud Infrastructure Security](/docs/cloud-infrastructure-security/), [Cloud SIEM](/docs/cse/), and [Cloud SOAR](/docs/cloud-soar/) are Sumo Logic's primary security solutions. But Sumo Logic offers additional security features, including apps, customizable dashboards, and tools to analyze your security data. These allow you to use Sumo Logic's core functionality, including data collection, ingestion, and storage, to produce findings that help protect your attack surfaces from threats.
 
-To get the most out of Cloud Infrastructure Security, use apps custom-built for the solution, like the [Cloud Infrastructure Security for AWS app](/docs/integrations/amazon-aws/cloud-infrastructure-security-for-aws/). This app provides visibility into your AWS environment to give you insights into active threats, security control failures, and suspicious activity. 
-
-You can use Cloud Infrastructure Security for:
-* [Security data lake](/docs/cloud-infrastructure-security/data-lake/). Store your structured and unstructured data in a centralized repository so it can be easily accessed for analytics.
-* [Audit and compliance](/docs/cloud-infrastructure-security/audit-and-compliance/). Audit your systems to ensure they are in compliance with the rules set by governments and regulatory organizations. This helps protect end users and keep information private and secure, in addition to outlining protocols in the event of a breach.
-* [Threat detection and investigation](/docs/cloud-infrastructure-security/threat-detection-and-investigation/). Identify and explore threats or security-related events within your assets, applications, or networks as quickly and effectively as possible.
-* [Application security](/docs/cloud-infrastructure-security/application-security/). Monitor your software development pipeline to ensure it is secure during the continuous integration/continuous deployment (CI/CD) process and production cycle.
-
-:::tip
-To learn more about Cloud Infrastructure Security and receive training certification, take the Cloud Infrastructure Security course in our [Sumo Logic Training Portal](/docs/get-started/training-certification-faq).
-:::
+You can use Sumo Logic for:
+* [Threat detection and investigation](/docs/additional-security-features/threat-detection-and-investigation/). Identify and explore threats or security-related events within your assets, applications, or networks as quickly and effectively as possible.
+* [Security data lake](/docs/additional-security-features/data-lake/). Store your structured and unstructured data in a centralized repository so it can be easily accessed for analytics.
```

Comment:
```suggestion
* [Security data lake storage](/docs/additional-security-features/data-lake/). Store your structured and unstructured data in a centralized repository so it can be easily accessed for analytics.
```